bin/server

```diff
@@ -47,7 +47,7 @@ async function init() {
 
         return logger.info('Server running at %s', server.info.uri);
     } catch (err) {
-        logger.error(err);
+        logger.error(err.toString());
```

Comment:
How about `JSON.stringify`, does JSON.stringify capture many info nosies?

Reply:
this is a bug with logger we use `winston`  This is what we should do long term https://github.com/winstonjs/winston/issues/1338#issuecomment-843793731